### DISCUSSION TITLE
Add Notes Field as a table cell to the Meeting List

### DISCUSSION THREAD

**Discussion by timmcdaniels** [code4recovery/12-step-meeting-list#1440] [Ideas]
It would be great to have the Notes field added to the Meeting List page: https://aaheartofva.org/meetings/

By the way, the FAQ mentions the info below, but I don't think the class is being added:

"Rows that have meeting notes will have a ‘notes’ class."

If the Notes field were added as a table cell on the list page, the content could be restricted to a certain number of words or characters with a "(more)" indicator and a link to the meeting.

**Comment by anchovie91471** [comment]
The "notes" class was added as as stopgap during covid and is no longer supported. We will definitely continue to evaluate the design of the meetings page based on user feedback. Thank you for your input!

**Comment by timmcdaniels** [comment]
Thanks for the consideration! We have a large number of users who only view the list view and miss Notes entirely.
